apps/explorer/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -115,7 +115,8 @@ async function getRecentTransactions(
                 endGatewayTxSeqNumber
             )
             .then((res: GetTxnDigestsResponse) =>
-                getDataOnTxDigests(network, res)
+                // results returned by getTransactionDigestsInRange is in ascending order
+                getDataOnTxDigests(network, res.reverse())
```

Comment:
`reverse` changes items in-place, can you use `[...res].reverse()` so that you copy the array before reversing?